src/Symfony/Component/Form/Extension/Core/Type/TextType.php

```diff
@@ -37,6 +37,7 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
             'compound' => false,
+            'accept_multiple_values' => false,
```

Comment:
@xabbuh this should set to true until 5.0 to avoid the bc break which was introduce in 3.4.21, else applications which did work before 3.4.21 still would crash.